src/h5web/visualizations/shared/AxisSystemProvider.tsx

```diff
@@ -1,28 +1,24 @@
 import React, { createContext, ReactElement, ReactNode } from 'react';
-import { AxisDomains } from './models';
+import { AxisConfig } from './models';
 
 export interface DisplayAxisProps {
-  axisDomains: AxisDomains;
-  showGrid?: boolean;
-  hasXLogScale?: boolean;
-  hasYLogScale?: boolean;
+  abscissaConfig: AxisConfig;
+  ordinateConfig: AxisConfig;
 }
 
-export const AxisSystemContext = createContext<DisplayAxisProps>({
-  axisDomains: { x: [0, 1], y: [0, 1] },
-});
+export const AxisSystemContext = createContext<DisplayAxisProps>(
+  {} as DisplayAxisProps
```

Comment:
I prefer the default state to be blatantly invalid and throw if used than to be valid and give us unexpected results silently.